cmd/manager/main.go

```diff
@@ -18,6 +18,9 @@ package main
 
 import (
 	"flag"
+
+	"k8s.io/klog"
```

Comment:
Nit: There's not really any need for these extra spaces. Normally I try to organize it like so:
```
Builtin Golang Packages

External Packages

Internal Packages
```

But that's not a hard fast rule at all

Reply:
It's a good point, and I'm usually a stickler for formatting too. I was thinking a separate PR to do that in all of our go files for the project.

Reply:
Sounds good. Let's merge this then.